Gemfile

```diff
@@ -8,10 +8,11 @@ ruby '2.5.1'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.1'
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3'
+#gem 'sqlite3'
```

Comment:
@IgorKrv why this block was commented on?

Reply:
uncommented